platform/android/export/export.cpp

```diff
@@ -744,6 +744,34 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+	//Needs to match the method parameters of EditorExportSaveSharedObject
+	static Error ignore_so_file(void *p_userdata, const SharedObject &p_so) {
```

Comment:
The function name `ignore_so_file` is kind of confusing. You mean you're ignoring the settings? But actually, you're include every .so file?